python/ambassador/config/resourcefetcher.py

```diff
@@ -866,15 +866,19 @@ def handle_k8s_service(self, k8s_object: AnyDict) -> HandlerResult:
                 objects = parse_yaml(annotations)
 
                 for obj in objects:
+                    if not obj:
+                        self.logger.warning(f"empty YAML document found in ambassador service: {resource_name}")
```

Comment:
How about `{resource_name}.{resource_namespace}` for consistency?

Reply:
Done.